DogeNet.WebApi/Startup.cs

```diff
@@ -0,0 +1,110 @@
+using DogeNet.DAL;
+using IdentityServer4.AccessTokenValidation;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DogeNet.WebApi
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            AppConfiguration = configuration;
+        }
+
+        public IConfiguration AppConfiguration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication(options =>
+            {
+                options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+            })
+                .AddIdentityServerAuthentication(options =>
+                {
+                    options.ApiName = "DogeNetWebAPI";
+                    options.Authority = AppConfiguration.GetValue<string>("Services:IdentityServer");
+                    options.RequireHttpsMetadata = false;
+                });
+
+            var connectionString = AppConfiguration.GetConnectionString(nameof(DBContext));
+            services.AddDbContext<DBContext>(options => options.UseSqlServer(connectionString));
+
+            services.AddControllers();
+            services.AddSwaggerGen(options =>
+            {
+                options.AddSecurityDefinition("oauth2", new OpenApiSecurityScheme
+                {
+                    Type = SecuritySchemeType.OAuth2,
+                    Flows = new OpenApiOAuthFlows
+                    {
+                        Password = new OpenApiOAuthFlow
+                        {
+                            TokenUrl = new Uri("https://localhost:10001/connect/token"),
+                            Scopes = new Dictionary<string, string>
+                            {
+                                {"DogeNetWebAPI", "DogeNet WebAPI"}
```

Comment:
Each const string should be moved to settings

Reply:
![image](https://user-images.githubusercontent.com/77879337/142200774-3eec3a83-b463-4a35-a361-26b9ead55882.png)
@oleh-leobit 
like that?